src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -50,39 +74,19 @@ private Appeal buildAppealFromData(Map<String, Object> pairs) {
         .build();
     }
 
-    private Appellant buildAppellant(Map<String, Object> pairs) {
-        return Appellant.builder()
-            .name(buildAppellantName(pairs))
-            .address(buildAppellantAddress(pairs))
-            .identity(buildAppellantIdentity(pairs))
-            .contact(buildAppellantContact(pairs))
-            .isAppointee(buildIsAppointeeField(pairs))
-        .build();
-    }
-
-    private String buildIsAppointeeField(Map<String, Object> pairs) {
-        if (areBooleansValid(pairs, "is_appointee", "is_not_appointee") && !doBooleansContradict(pairs, "is_appointee", "is_not_appointee")) {
-            return convertBooleanToYesNoString((boolean) pairs.get("is_appointee"));
-        }
-        return null;
-    }
-
     private Representative buildRepresentative(Map<String, Object> pairs) {
-        boolean doesRepExist = findBooleanExists(getField(pairs,"representative_person_title"), getField(pairs,"representative_person_first_name"),
-            getField(pairs,"representative_person_last_name"), getField(pairs,"representative_address_line1"), getField(pairs,"representative_address_line2"),
-            getField(pairs,"representative_address_line3"), getField(pairs,"representative_address_line4"), getField(pairs,"representative_postcode"),
-            getField(pairs,"representative_phone_number"), getField(pairs,"representative_name"));
+        boolean doesRepExist = hasPerson(pairs, "representative");
 
         if (doesRepExist) {
             return Representative.builder()
                 .hasRepresentative(convertBooleanToYesNoString(doesRepExist))
-                .name(buildRepresentativePersonName(pairs))
-                .address(buildRepresentativeAddress(pairs))
-                .organisation(getField(pairs,"representative_name"))
-                .contact(buildRepresentativeContact(pairs))
+                .name(buildPersonName(pairs, "representative"))
```

Comment:
representative can be extracted to a constant ?

Reply:
done